IsraelHiking.Web/src/application/services/route-statistics.service.ts

```diff
@@ -107,22 +105,27 @@ export class RouteStatisticsService {
                 (currentPoint.coordinate[0] - prevPoint.coordinate[0]);
         }
 
-        let pts = resample(routeStatistics.points.map(p=>p.coordinate), { dist: 0.025 }, false);
+	    // calculate total gain & loss:
+        // resample coordinates along route at uniform resolution
+        let coordinates = routeStatistics.points.map(p=>[p.coordinate[0], p.coordinate[1]]);
```

Comment:
`coordinate` is already an array of two elements, this can simply be written:
`routeStatistics.points.map(p => p.coordinate)`